### DISCUSSION TITLE
CMYK jpg convert colors

### DISCUSSION THREAD

**Discussion by tjrunt** [ImageMagick/ImageMagick#3001] [General] [answered]
Hello 

First of all thanks in advance for any help. 

Now I dont know if this is a bug with the windows version of Imagemagick or something else.

Also my current issue sounds similar to the following issue posted several years back but I cant view the posted example images so I'm not 100%.

https://legacy.imagemagick.org/discourse-server/viewtopic.php?t=24412

What I am looking to do is convert CMYK jpg's to tiff's. Now when I convert them on the Mac it works as expected but when I do the same thing on windows which is ultimately where the conversions will be taking place the converted images colors are inverted.  The output is the same weather I convert to the CMYK jpgs to png, tiff, eps, jpg ect...

the command I'm using is:

convert CMYK_imageName_in.jpg CMYK_imageName_out.tif

The ImageMagick version info in the 2 systems are:

PC  

Version: ImageMagick 7.0.10-37 Q16 x64 2020-11-07 http://www.imagemagick.org
Copyright: Copyright (C) 1999-2018 ImageMagick Studio LLC
License: http://www.imagemagick.org/script/license.php
Visual C++: 192729112
Features: Cipher DPC HDRI Modules OpenCL OpenMP(2.0)
Delegates (built-in): bzlib cairo flif freetype gslib heic jng jp2 jpeg lcms lqr lzma openexr pangocairo png ps raw
 rsvg tiff webp xml zlib

MAC

Version: ImageMagick 7.0.10-0 Q16 x86_64 2020-03-08 https://imagemagick.org
Copyright: © 1999-2020 ImageMagick Studio LLC
License: https://imagemagick.org/script/license.php
Features: Cipher DPC HDRI Modules OpenMP(3.1) 
Delegates (built-in): bzlib freetype heic jng jp2 jpeg lcms ltdl lzma openexr png tiff webp xml zlib

[jpg_conversions.zip](https://github.com/ImageMagick/ImageMagick/files/5711335/jpg_conversions.zip)

**Comment by snibgo** [comment]
Your input CMYK_test_monkey_org.jpg is CMYK, with embedded profile.

PNG can't store CMYK images. If you write to PNG, IM will convert the CMYK to sRGB. But it does so in a too-simple way. If you want sRGB, I suggest you convert it with "-profile sRGB.icc" or similar.

Your command creates CMYK_imageName_out.tif, but you didn't include that file in your zip. Why not?

This command:
```
magick CMYK_test_monkey_org.jpg x.tif
```
... with IM v7.0.8-64 or v7.0.10-45 works fine, creating x.tif, a CMYK image with embedded profile.

**Comment by tjrunt** [comment, marked as answer]
I ended up doing a reinstall and that solved the issue. Something must have corrupted on the original install that only effected cmyk jpegs
